src/components/AuthSettings.tsx

```diff
@@ -134,7 +141,7 @@ export default function AuthSettings(): ReactElement {
                   <button
                     type="button"
                     className="bg-white py-2 px-4 border border-gray-300 rounded-md shadow-sm text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-brand-lighter"
-                    onClick={() => props.resetForm({ values: initalFormData })}
+                    onClick={() => createFolder()}
```

Comment:
Use the above renamed function.

Reply:
Actually, I was just looking at this Reset functionality, and what it is doing now is not correct. It looks like we are deleting the access key and secret, then saving it. Before, it was just resetting the form to what it was, prior to any changes.

Ultimately, the Reset button will go away, and most likely the Save button (depending on how we handle the profile saving).

The goal is to get the single access key / secret functionality removed, and replace it with the profile system. Each profile could possibly be a different type, but for now, all we have is the Storj DCS type.